### PR TITLE
Music pause canvas

### PR DESCRIPTION
**Pull request by JoshuaAtherton**
- scene manager has a new menu class that controls the music which scene manager controls
- Added clickable images for playing and pausing background audio and sound fx
   -sound engine fx methods not implemented for these but they will work once done
- Added an orange city background

### CODE REVIEW

File: public/js/SceneManager.js [hunk 1]
```diff
@@ -52,9 +53,96 @@ class SceneManager {
...
+      }
+      //fx x
+      if (
+        clickXandY.x >= this.x + 10 &&
+        clickXandY.x <= this.x + 70 &&
+        clickXandY.y >= this.y + 30 &&
+        clickXandY.y <= this.y + 50
+      ) {
+        changeInState = true;
+        this.xFxChecked = this.xFxChecked ? false : true;
```
Comment (steveng9) [inline public/js/SceneManager.js]: could you just make these `this.xFxChecked = !this.xFxChecked`?, although I do like the ternary operator:)

File: public/js/SceneManager.js [hunk 2]
```diff
@@ -52,9 +53,96 @@ class SceneManager {
...
+  }
+
+  draw() {
+    //draw music music menu background
+    this.ctx.drawImage(this.menu, this.x, this.y);
+    if (this.xMusicChecked) {
+      this.ctx.drawImage(this.xMusic, this.x, this.y);
+    }
+    if (this.xFxChecked) {
+      this.ctx.drawImage(this.xFx, this.x, this.y);
```
Comment (steveng9) [inline public/js/SceneManager.js]: The clicking boxes are not quite lined up correctly. When I click in the upper left quadrant of the box, no change, when I click outside of the box to the lower right it 'checks' when it shouldn't.

File: public/style.css [hunk 3]
```diff
@@ -4,7 +4,10 @@ canvas {
     margin: auto;
     display: block;
     width: auto; /* could specify canvas element width here */
+    cursor: crosshair;
```
Comment (steveng9) [inline public/style.css]: Love this!!

**Comment by steveng9** [comment]
I really like the position and the style of the music menu! I also think the curser is perfect. 

Could we finish up integrating sound FX options before merging, and lining up the mouse clicks, just so we can be done with it and move on after merging?

File: public/js/SceneManager.js (hunk 1 above)
Comment (JoshuaAtherton) [reply public/js/SceneManager.js]: That makes more sense. I'll change it up

Comment (JoshuaAtherton) [reply public/js/SceneManager.js]: Well I'm not sure now we can figure it out tonight lol

**Comment by JoshuaAtherton** [comment]
It should be good to merge in now. As as side note so I don't forget how do you guys feel about S being for crouch and I tried out a different hurt noise?

**Pull request merged**